pkg/models/dhcp_server.go

```diff
@@ -0,0 +1,72 @@
+package models
+
+import (
+	"encoding/json"
+	"time"
+)
+
+type CreateNetworkDhcpServerRequest struct {
+	NetworkDhcpServer CreateNetworkDhcpServer `json:"networkDhcpServer"`
+}
+
+type CreateNetworkDhcpServer struct {
+	ID              int         `json:"-" tf:"id,computed"`
+	NetworkServerID int         `json:"networkServerId" tf:"network_server_id,computed"`
+	Name            string      `json:"name" tf:"name"`
+	LeaseTime       int         `json:"leaseTime" tf:"lease_time"`
+	ServerIPAddress string      `json:"serverIpAddress" tf:"server_address"`
+	Config          *DhcpConfig `json:"config" tf:"config,sub"`
+}
+
+type DhcpConfig struct {
+	EdgeCluster string `json:"edgeCluster"  tf:"edge_cluster"`
```

Comment:
i think you need to pass the edge cluster provider id here. Hope you have required data sources created for it.

Reply:
yes its der